Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -361,6 +376,10 @@ class ITKIOGDCM_EXPORT GDCMImageIO : public ImageIOBase
 
   bool m_LoadPrivateTags;
 
+  bool m_YBRtoRGB;
+
+  bool m_RGBtoYBRBeforeRLECompression;
+
```

Comment:
Y'CbCr is a right way to go with RLE, makes it more efficient, ~10%-20% at least and a way to save Y'CbCr explicit, if required.